libs/cypress-indexeddb/src/lib/object-store-CRUD.ts

```diff
@@ -76,6 +122,17 @@ export function updateItem(
   return setItem('update', store, key, value);
 }
 
+/**
+ * Add item for provided store key
+ *
+ * @remarks The `addItem` method adds the value provided to the store. You can chain assertions from this method. If you use TypeScript, you can set the type of the returned value
+ *
+ * @param store `IDBObjectStore` instance
+ * @param value item value
+ *
+ * @returns IDBObjectStore
+ * @throws {Error} If the connections fails to open or the read operation fails.
```

Comment:
technically it only throws an error if it is chained off from a method that does not return an object store.